docs/content/Link.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Link
+status: Alpha
+---
+
+import {PropsTable} from '../src/props-table'
+
+The Link component styles anchor tags with default hyperlink color cues and hover text decoration. `Link` is used for destinations, or moving from one page to another.
+
+In special cases where you'd like a `<button>` styled like a `Link`, use `<Link as='button'>`. Make sure to provide a click handler with `onClick`.
+
+**Important:** When using the `as` prop, be sure to always render an accessible element type, like `a`, `button`, `input`, or `summary`.
+
+## Default example
+
+```jsx live
+<Link sx={{mb: 1}} href="https://github.com">
+  Link
+</Link>
+```
+
+## Component props
+
+<PropsTable>
+  <PropsTable.Row name="href" type="String" description="URL to be used for the Link" />
+  <PropsTable.Row
+    name="muted"
+    type="Boolean"
+    defaultValue="false"
+    description="Uses a less prominent shade for Link color, and the default link shade on hover"
+  />
+  <PropsTable.Row name="underline" type="Boolean" defaultValue="false" description="Adds underline to the Link" />
+  <PropsTable.Row name="hoverColor" type="String" description="Color used when hovering over link" />
```

Comment:
Nit: In typescript, in `string` and `boolean` are lowercase, right?

```suggestion
  <PropsTable.Row name="href" type="string" description="URL to be used for the Link" />
  <PropsTable.Row
    name="muted"
    type="boolean"
    defaultValue="false"
    description="Uses a less prominent shade for Link color, and the default link shade on hover"
  />
  <PropsTable.Row name="underline" type="boolean" defaultValue="false" description="Adds underline to the Link" />
  <PropsTable.Row name="hoverColor" type="string" description="Color used when hovering over link" />
```

Reply:
yes! At least, the lowercase types are the ones you want 99% of the time. Good catch!